site-manage/site-manage-impl/impl/src/java/org/sakaiproject/sitemanage/impl/SiteManageServiceImpl.java

```diff
@@ -111,9 +112,21 @@ public boolean importToolsIntoSiteThread(final Site site, final List<String> exi
         Runnable siteImportTask = () -> {
             sessionManager.setCurrentSession(session);
             sessionManager.setCurrentToolSession(toolSession);
-            eventTrackingService.post(eventTrackingService.newEvent(SiteService.EVENT_SITE_IMPORT_START, site.getReference(), false));
-
-            try {
+            
+			String importSites ="";
+			for (Map.Entry<String, List<String>> entry : importTools.entrySet()) {
+				if(importSites.length() >= 255)
+					break;
```

Comment:
Please use braces on single line conditional statements.

Reply:
Done

Reply:
I think maybe you forgot to push your changes here; this `if` statement does not have braces.